package.json

```diff
@@ -1,990 +1,989 @@
-          "type": "string",
-          "description": "The shell command to get current input method.",
-          "default": "/path/to/im-select"
-        },
-        "vim.whichwrap": {
-          "type": "string",
-          "description": "Comma-separated list of motion keys that should wrap to next/previous line.",
-          "default": ""
-        },
-        "vim.report": {
-          "type": "number",
-          "description": "Threshold for reporting number of lines changed.",
-          "default": 2,
-          "minimum": 1
-        },
-        "vim.digraphs": {
-          "type": "object",
-          "description": "Custom digraph shortcuts for inserting special characters, expressed as UTF16 code points.",
-          "default": {}
-        },
-        "vim.wrapscan": {
-          "type": "boolean",
-          "description": "Searches wrap around the end of the file.",
-          "default": true
-        },
-        "vim.startofline": {
-          "type": "boolean",
-          "markdownDescription": "When `true` the commands listed below move the cursor to the first non-blank of the line.  When `false` the cursor is kept in the same column (if possible).  This applies to the commands: `<C-d>`, `<C-u>`, `<C-b>`, `<C-f>`, `G`, `H`, `M`, `L`, `gg`, and to the commands `d`, `<<` and `>>` with a linewise operator.",
-          "default": true
-        }
-      }
-    },
-    "languages": [
-      {
-        "id": "vimrc",
-        "extensions": [
-          ".vimrc"
-        ]
-      }
-    ],
-    "grammars": [
-      {
-        "language": "vimrc",
-        "scopeName": "source.vimrc",
-        "path": "./syntaxes/vimrc.tmLanguage.json"
-      }
-    ]
-  },
-  "scripts": {
-    "vscode:prepublish": "gulp build",
-    "build": "gulp build",
-    "test": "node ./out/test/runTest.js",
-    "forceprettier": "gulp forceprettier"
-  },
-  "dependencies": {
-    "diff-match-patch": "1.0.5",
-    "lodash": "^4.17.15",
-    "neovim": "4.9.0",
-    "untildify": "4.0.0",
-    "winston": "3.2.1",
-    "winston-console-for-electron": "0.0.6",
-    "winston-transport": "4.3.0"
-  },
-  "devDependencies": {
-    "@types/diff": "4.0.2",
-    "@types/diff-match-patch": "1.0.32",
-    "@types/glob": "7.1.1",
-    "@types/lodash": "4.14.153",
-    "@types/mocha": "7.0.2",
-    "@types/node": "12.12.31",
-    "@types/sinon": "9.0.4",
-    "@types/vscode": "1.37.0",
-    "gulp": "4.0.2",
-    "gulp-bump": "3.1.3",
-    "gulp-git": "2.10.1",
-    "gulp-sourcemaps": "2.6.5",
-    "gulp-tag-version": "1.3.1",
-    "gulp-tslint": "8.1.4",
-    "gulp-typescript": "5.0.1",
-    "minimist": "1.2.5",
-    "mocha": "7.2.0",
-    "plugin-error": "1.0.1",
-    "prettier": "2.0.5",
-    "sinon": "9.0.2",
-    "ts-loader": "7.0.5",
-    "tslint": "6.1.2",
-    "typescript": "3.9.3",
-    "vscode-test": "1.4.0",
-    "webpack": "4.43.0",
-    "webpack-cli": "3.3.11",
-    "webpack-merge": "4.2.2",
-    "webpack-stream": "5.2.1"
-  }
+	"name": "vim",
```

Comment:
Could you please run `gulp forceprettier` to get rid of this diff?

Reply:
`npx prettier package.json`, actually - before just now prettier wasn't configured to touch json files

Reply:
@J-Fields both commands don't affect `package.json` at all